src/workflow/trinary_input.c

```diff
@@ -35,12 +38,28 @@ static void _confirm(const char* word, void* param)
     workflow_blocking_unblock();
 }
 
+static void _select(uint8_t idx)
+{
+    ui_screen_stack_pop();
+    if (idx == 0) {
+        _cancel_reason = WORKFLOW_TRINARY_INPUT_RESULT_DELETE;
+        workflow_cancel_force();
+    } else if (idx == 1) {
```

Comment:
It feels a bit fragile, referring to indexes of an array in another function.
How about making `words[] = {"Edit prev", "Cancel restore"}` and these indexes constants, outside of the `_select` and `_cancel`?

Reply:
Or, at the very least, add a comment explaining which array `idx` refers to?

Reply:
done